src/main/java/hudson/plugins/timestamper/TimestampNote.java

```diff
@@ -104,4 +113,11 @@ public Timestamp getTimestamp(Run<?, ?> build) {
     }
     return null; // each time-stamp note affects one line only
   }
+
+  public void plain(OutputStream out) throws IOException {
+    TimestamperConfig config = GlobalConfiguration.all().get(TimestamperConfig.class);

```

Comment:
What would happen if the user change the configuration? I think we won't see the changes.